app/lib/common/widget/like_button.dart

```diff
@@ -0,0 +1,176 @@
+import 'package:effektio/blocs/like_animation.dart';
+import 'package:effektio/common/store/separatedThemes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class LikeButton extends StatefulWidget {
```

Comment:
I would rather like to use packages for achieving animations in this regard but its okay if it doesn't fulfill the required one.

Reply:
I didn't find such a package for this animation so I built this.